frame/contracts/src/weights.rs

```diff
@@ -173,772 +173,794 @@ pub trait WeightInfo {
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	/// Storage: Contracts DeletionQueue (r:1 w:0)
-	/// Proof: Contracts DeletionQueue (max_values: Some(1), max_size: Some(16642), added: 17137, mode: Ignored)
+	/// Proof: Contracts DeletionQueue (max_values: Some(1), max_size: Some(16642), added: 17137, mode: Measured)
 	fn on_process_deletion_queue_batch() -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `109`
-		//  Estimated: `0`
-		// Minimum execution time: 2_564 nanoseconds.
-		Weight::from_ref_time(2_722_000)
-			.saturating_add(Weight::from_proof_size(0))
+		//  Estimated: `604`
+		// Minimum execution time: 2_522 nanoseconds.
+		Weight::from_ref_time(2_752_000)
+			.saturating_add(Weight::from_proof_size(604))
 			.saturating_add(T::DbWeight::get().reads(1_u64))
 	}
 	/// Storage: Skipped Metadata (r:0 w:0)
-	/// Proof Skipped: Skipped Metadata (max_values: None, max_size: None, mode: Ignored)
+	/// Proof Skipped: Skipped Metadata (max_values: None, max_size: None, mode: Measured)
 	/// The range of component `k` is `[0, 1024]`.
 	fn on_initialize_per_trie_key(k: u32, ) -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `481 + k * (69 ±0)`
-		//  Estimated: `0`
-		// Minimum execution time: 10_292 nanoseconds.
-		Weight::from_ref_time(7_474_496)
-			.saturating_add(Weight::from_proof_size(0))
-			// Standard Error: 901
-			.saturating_add(Weight::from_ref_time(956_864).saturating_mul(k.into()))
+		//  Measured:  `343`
+		//  Estimated: `343`
+		// Minimum execution time: 5_283 nanoseconds.
+		Weight::from_ref_time(6_210_922)
+			.saturating_add(Weight::from_proof_size(343))
+			// Standard Error: 62
+			.saturating_add(Weight::from_ref_time(2_370).saturating_mul(k.into()))
 			.saturating_add(T::DbWeight::get().reads(1_u64))
-			.saturating_add(T::DbWeight::get().writes(1_u64))
-			.saturating_add(T::DbWeight::get().writes((1_u64).saturating_mul(k.into())))
```

Comment:
This completely brokes down as if no work is performed anymore. Neither does `k` add a write (which is should) nor is there a substantial amount of weight attached to `k`. Need to investigate. This is not the case on master.